specs/lib/removeUnusedProgramsSpec.js

```diff
@@ -0,0 +1,175 @@
+'use strict';
+
+var removeUnusedPrograms = require('../../').removeUnusedPrograms;
+var OptimizationStatistics = require('../../').OptimizationStatistics;
+
+describe('removeUnusedPrograms', function() {
+    it('removes a program', function() {
+        var gltf = {
+            "programs": {
+                "program_0": {
+                    "attributes": [
+                        "a_normal",
+                        "a_position",
+                        "a_texcoord0"
+                    ],
+                    "fragmentShader": "CesiumTexturedBoxTest0FS",
+                    "vertexShader": "CesiumTexturedBoxTest0VS"
+                },
+                "unusedProgramId": {
+                    "attributes": [
+                        "a_normal",
+                        "a_position",
+                        "a_texcoord0"
+                    ],
+                    "fragmentShader": "CesiumTexturedBoxTest0FS",
+                    "vertexShader": "CesiumTexturedBoxTest0VS"
+                }
+            },
+            "techniques": {
+                "technique0": {
+                    "attributes": {
+                        "a_normal": "normal",
+                        "a_position": "position",
+                        "a_texcoord0": "texcoord0"
+                    },
+                    "parameters": {
```

Comment:
Likewise, we can simplify `parameters` here and `uniforms` below by just keep `modelViewMatrix` and `projectionMatrix`.

These are just snippets so they don't need everything.